config/fi_provider.m4

```diff
@@ -127,8 +127,15 @@ AC_DEFUN([FI_PROVIDER_SETUP],[
 		 AS_IF([test -f "$srcdir/prov/$1/include/rdma/fi_direct.h"],
 			[AC_MSG_RESULT(yes)],
 			[AC_MSG_RESULT(no)
-			  AC_MSG_ERROR([$1 provider was requested as direct, but is missing fi_direct.h])]
-			)])
+			 AC_MSG_CHECKING(for $srcdir/prov/$1/include/rdma/fi_direct.h.in)
+			 AS_IF([test -f "$srcdir/prov/$1/include/rdma/fi_direct.h.in"],
+				[AC_MSG_RESULT(yes)],
+				[AC_MSG_RESULT(no)
+				 AC_MSG_ERROR([$1 provider was requested as direct, but is missing fi_direct.hand fi_direct.h.in])]
```

Comment:
Hi Jeff - Regarding formatting / whitespace issues is there some sort of source formatting tool that libfabric has I could use to run on these files that would get things into the right style?